tools/anthosbm-ansible-module/roles/anthos/tasks/main.yml

```diff
@@ -14,49 +14,77 @@
 
 - name: Reset ssh connection to allow group changes
   meta: reset_connection
-  
-- name: Delete bmctl workspace cluster directory if exists
```

Comment:
What is the reason for removing this?

Reply:
Not sure, I don't think it's an issue to leave in but wouldn't mind confirmation from @avinashkumar1289

Reply:
Not sure, I don't think it's an issue to leave in but wouldn't mind confirmation from @avinashkumar1289

Reply:
@ravi-dalal  there was an instance where the client had accidentally run the create playbook after the cluster was created. Because of that delete bmctl workspace, the kubeconfig file was deleted. We were basically not clearing anything unless the user needs to

Reply:
We should backup the kubeconfig file at another location as part of this package. I do not remember why I added delete task but I think I ran into some issues. Let's remove it but also backup the kubeconfig at a configuration location.

Reply:
Yeah so if the workspace is already created, this playbook will fail.. I think thats why we are deleting the workspace if it exists ..Yes backing up Kubeconfig is something needs to be done

Reply:
I've added a note to include a backup kubeconfig file (in a future commit) and will resolve this as is for now.